### PR TITLE
bump elixir webrtc to 0.3

### PR DESCRIPTION
**Pull request by mat-hek**
Before merging, we need to figure out the ex_sdp version conflicts, see https://github.com/membraneframework/ex_sdp/pull/50

### CODE REVIEW

**Review by mickel8 — APPROVED** [review]
LGTM but I just want to make sure that you know that we introduced MediaStreams in 0.3 and right now, your implementation alaways adds tracks separately. So on the FE in `ontrack` callback, those tracks won't have any streams assigned. 

Your examples will work as you do `videoPlayer.srcObject.addTrack` but I am just mentioning. 

You can read more here:  https://github.com/elixir-webrtc/ex_webrtc/releases/tag/v0.3.0

**Comment by FelonEkonom** [comment]
@mickel8 I have implemented handling keyframe requests since your last review, could you review my changes?

File: lib/membrane_webrtc/ex_webrtc/sink.ex [hunk 1]
```diff
@@ -133,6 +133,38 @@ defmodule Membrane.WebRTC.ExWebRTCSink do
     raise "Track #{inspect(track_id)} was rejected by the other peer"
   end
 
+  @impl true
+  def handle_info({:ex_webrtc, _from, {:rtcp, rtcp_packets}}, ctx, state) do
+    request_keyframe? =
+      rtcp_packets
+      |> Enum.reduce(false, fn packet, request_keyframe? ->
+        case packet do
+          {_track_id, %ExRTCP.Packet.PayloadFeedback.PLI{}} ->
```
Comment (mickel8) [inline lib/membrane_webrtc/ex_webrtc/sink.ex]: I think I would make sure that track_id refers to video track. Just to be sure

File: lib/membrane_webrtc/ex_webrtc/sink.ex [hunk 2]
```diff
@@ -133,6 +133,38 @@ defmodule Membrane.WebRTC.ExWebRTCSink do
...
+            Membrane.Logger.debug("Keyframe request received: #{inspect(packet)}")
+            true
+
+          _other ->
+            Membrane.Logger.debug("Ignoring RTCP packet: #{inspect(packet)}")
```
Comment (mickel8) [inline lib/membrane_webrtc/ex_webrtc/sink.ex]: This might be pretty verbose

File: lib/membrane_webrtc/ex_webrtc/sink.ex [hunk 3]
```diff
@@ -133,6 +133,38 @@ defmodule Membrane.WebRTC.ExWebRTCSink do
...
+            request_keyframe?
+        end
+      end)
+
+    actions =
+      if request_keyframe? do
+        ctx.pads
+        |> Enum.flat_map(fn {pad_ref, pad_data} ->
```
Comment (mickel8) [inline lib/membrane_webrtc/ex_webrtc/sink.ex:162]: Why sending keyframe on all video pads? Just send on a pad that refers to `track_id` from line 142

File: lib/membrane_webrtc/ex_webrtc/sink.ex [hunk 4]
```diff
@@ -133,6 +133,38 @@ defmodule Membrane.WebRTC.ExWebRTCSink do
...
+          case pad_data.options.kind do
+            :video -> [event: {pad_ref, %Membrane.KeyframeRequestEvent{}}]
```
Comment (mat-hek) [inline lib/membrane_webrtc/ex_webrtc/sink.ex:164]: Shouldn't we somehow 'debounce' the keyframe requests? AFAIR the browser is going to spam us with PLIs when it's unable to decode the stream to make sure we receive a PLI ASAP. And we shouldn't generate as many keyframes as PLIs. cc @mickel8

File: lib/membrane_webrtc/ex_webrtc/sink.ex (hunk 3 above)
Comment (FelonEkonom) [reply lib/membrane_webrtc/ex_webrtc/sink.ex:162]: In fact, there is no`track_id` there

File: lib/membrane_webrtc/ex_webrtc/sink.ex (hunk 1 above)
Comment (FelonEkonom) [reply lib/membrane_webrtc/ex_webrtc/sink.ex]: In fact, there is no `track_id` here

File: lib/membrane_webrtc/ex_webrtc/sink.ex (hunk 4 above)
Comment (mickel8) [reply lib/membrane_webrtc/ex_webrtc/sink.ex:164]: Sounds resonable but how to distinct between PLI retransmission and real PLI :thinking:

**Pull request merged**